kcidb/orm/data.py

```diff
@@ -344,10 +344,14 @@ def format_dot(self):
             field_json_schemas=dict(
                 id=_CHECKOUT['id'],
                 git_commit_hash=_CHECKOUT['git_commit_hash'],
+                git_commit_tags=_CHECKOUT['git_commit_tags'],
+                git_commit_message=_CHECKOUT['git_commit_message'],
                 patchset_hash=_CHECKOUT['patchset_hash'],
                 origin=_CHECKOUT['origin'],
                 git_repository_url=_CHECKOUT['git_repository_url'],
                 git_repository_branch=_CHECKOUT['git_repository_branch'],
+                git_repository_branch_tip=_CHECKOUT[
+                    'git_repository_branch_tip'],
```

Comment:
Could you put the closing bracket on the next line, on the same column the key starts at? This way it will be easier to notice.